examples/solvers/forward_backward_primal_dual_denoising.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Total variation denoising using the Forward-backward solver.
+
+Solves the optimization problem
+
+    min_{0 <= x <= 255}  1/2 ||x - g||_2^2 + lam || |grad(x)| ||_1
+
+where ``grad`` is the spatial gradient and ``g`` the given noisy data.
+"""
+
+import numpy as np
+import scipy.misc
+import odl
+import odl.solvers as odlsol
+
+# Parameters
+n = 256
+
+# Create a space
+space = odl.uniform_discr([0, 0], [n, n], [n, n])
+
+# Load image and noise
+data = space.element(np.rot90(scipy.misc.ascent()[::2, ::2], 3))
+noise = odl.phantom.white_noise(space) * 10.0
+
+# Create noisy data
+noisy_data = data + noise
+data.show('Original data')
+noisy_data.show('Noisy convolved data')
+
+
+# Gradient for TV regularization
+gradient = odl.Gradient(space)
+
+# Assemble all operators
+lin_ops = [gradient]
+
+# Create proximals as needed
```

Comment:
Same here, the comment does not really add anything. Should be specific to this example and the method.